src/acl/external/LDAP_group/required.m4

```diff
@@ -5,4 +5,6 @@
 ## Please see the COPYING and CONTRIBUTORS files for details.
 ##
 
-AC_CHECK_HEADERS([ldap.h winldap.h],[BUILD_HELPER="LDAP_group"])
+AS_IF([test "x$squid_cv_LDAP" != "x"],[
```

Comment:
I thought that when we use SQUID_CHECK_LIB_WORKS(ldap), we should test LIBLDAP_LIBS rather than squid_cv_LDAP to determine whether SQUID_CHECK_LIB_WORKS() has detected support for LDAP. Am I missing something?

Reply:
You are correct, changing